package.json

```diff
@@ -19,8 +19,8 @@
     "lodash": "~2.4.1",
     "async": "~0.9.0",
     "waterline-errors": "~0.10.0",
-    "waterline-sequel": "~0.0.2",
-    "waterline-cursor": "~0.0.3"
+    "waterline-sequel": "git://github.com/AnalogJ/waterline-sequel.git",
```

Comment:
> +    "waterline-sequel": "git://github.com/AnalogJ/waterline-sequel.git",

@AnalogJ Yea this is never going to happen. Just submit a PR to waterline-sequel and we'll merge the work in there.